src/array/utf8/mod.rs

```diff
@@ -579,3 +590,25 @@ impl<O: Offset> Default for Utf8Array<O> {
         Utf8Array::new(data_type, Default::default(), Default::default(), None)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_apply_validity() {
```

Comment:
The tests are in here: https://github.com/jorgecarleitao/arrow2/blob/main/tests/it/array/utf8/mod.rs, so that we also test that the functions are available outside the crate (and to avoid re-compiling the crate whenever we play with tests). Could you move this one there?

Reply:
Yep, done